### PR TITLE
Generate dune-project with action-plugin 0.1 if it does not exist

### LINKED ISSUE #27: consider adding dynamic-action-plugin to dune-project
Right now dune generates an empty dune-project and requests you do add `(using action-plugin 0.1)`.

Perhaps we should do this automatically.

### PR DESCRIPTION
**Pull request by coproduto**
fixes #27

### CODE REVIEW

File: jscomp/main/bsb_main.ml [hunk 1]
```diff
@@ -91,6 +91,37 @@ let bsb_main_flags : (string * spec * string) array =
...
+    close_in inp;
+    get_version line
+  with
+  | Unix.Unix_error(_, _, _) -> None
+
+let make_dune_project_if_does_not_exist proj_dir =
+  let dune_project = proj_dir // Literals.dune_project in
+  if Sys.file_exists dune_project then ()
+  else
+    let dune_version = get_dune_version () in
```
Comment (anmonteiro) [inline jscomp/main/bsb_main.ml:114]: let's not shell out to dune.

It should be safe to output `(lang dune 2.8)` (even 2.7 should be enough), as that's the lower bound for the features that we need.

**Review by anmonteiro — COMMENTED** [review]
This looks great! I think this is a net gain for developer experience, and I'm in favor.

There's only a minor thing we need to address, and it should be good to go!

File: jscomp/main/bsb_main.ml (hunk 1 above)
Comment (coproduto) [reply jscomp/main/bsb_main.ml]: Just added a commit removing the shelling out and hardcoding the version to 2.8.

**Review by EduardoRFS — APPROVED** [review]
anime girl gif being super happy because of a new contributor

**Pull request merged**